tooling/maven/camel-restdsl-openapi-plugin/src/main/java/org/apache/camel/maven/generator/openapi/AbstractGenerateMojo.java

```diff
@@ -33,6 +40,7 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import io.apicurio.datamodels.Library;
 import io.apicurio.datamodels.openapi.models.OasDocument;
+import io.swagger.v3.parser.core.models.AuthorizationValue;
```

Comment:
Can we avoid swagger imports, as we should ideally only use the datamodel from apicurio.

Reply:
I removed the openapi / swagger dependencies...